dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets.jsp

```diff
@@ -752,12 +751,15 @@
                     <input type="hidden" name="cmd" value="prepublish">
                     <div class="portlet-toolbar" style="height: 48px">
                         <div class="portlet-toolbar__actions-secondary">
+                            <div id="matchingResultsDiv" style="display: none" class="portlet-toolbar__info"></div>
+                        </div>
+                        <div class="portlet-toolbar__actions-search" style="width: 270px;">
+                            <input type="text" dojoType="dijit.form.TextBox" tabindex="1" placeholder="<%= LanguageUtil.get(pageContext, "Search").replace("\"", "'") %>" onKeyUp='doSearch()' name="allFieldTB" id="allFieldTB" value="<%=_allValue %>">

```

Comment:
Based on the Figma Mock, the placeholder should be `Type for search...` instead of just `Search`. So maybe we should create another language variable for that. We can add new language variables in this [file](https://github.com/dotCMS/core/blob/master/dotCMS/src/main/webapp/WEB-INF/messages/Language.properties)

<img width="329" alt="Screenshot 2023-01-05 at 11 25 43 AM" src="https://user-images.githubusercontent.com/72418962/210816336-dd2de9f8-953d-423d-a01a-eddfba85eaba.png">

Reply:
Noted

Reply:
updated